src/pages/iou/request/step/IOURequestStepScan/index.tsx

```diff
@@ -977,6 +978,7 @@ function IOURequestStepScan({
                                 receiptImageTopPosition={receiptImageTopPosition}
                             />
                         )}
+                        {!isMobile() && <DownloadAppBanner />}
```

Comment:
Generally, we should be avoiding using isMobile() checks and instead having a separate `index.web.ts` and `index.ts` file, no?

Reply:
Ok that makes sense to me. Can we add a comment explaining why we are using it here?